.github/workflows/build-images.yaml

```diff
@@ -79,3 +79,9 @@ jobs:
         run: |
           echo ${{ steps.docker_build.outputs.digest }}
           echo ${{ steps.docker_build_distroless.outputs.digest }}
+      - name: Logout from DockerHub
+        run: docker logout
+        if: github.event_name != 'pull_request'
+      - name: Logout from GitHub Container Registry
+        run: docker logout ghcr.io
+        if: github.event_name != 'pull_request'
```

Comment:
This isn't necessary here because of the following reasons,
1. We are using GitHub-hosted runners, which are ephemeral. each runner is a fresh instance for each job and is destroyed after completion. So, any credentials used during the job don't stick around.
2. Cleaning up resources isn't necessary since this is the final step, and there aren't any conflicting steps or jobs interacting with Docker Hub.

Reply:
> This isn't necessary here because of the following reasons,
> 
> 1. We are using GitHub-hosted runners, which are ephemeral. each runner is a fresh instance for each job and is destroyed after completion. So, any credentials used during the job don't stick around.
> 2. Cleaning up resources isn't necessary since this is the final step, and there aren't any conflicting steps or jobs interacting with Docker Hub.

@jayasanka-sack , actually i dont know about that, thanks for sharing that